### ISSUE TITLE
Clarify documentation: callback not required to create an image

### ISSUE THREAD

**Issue by stevage** [oliver-moran/jimp#957]
The documentation says:

> Iyou want to begin with an empty Jimp image, you can call the Jimp constructor passing the width and height of the image to create and a Node-style callback:

```
 new Jimp(256, 256, (err, image) => {
  // this image is 256 x 256, every pixel is set to 0x00000000
});
```

However, it works fine to not pass the callback:

```js
const image = new Jimp(256, 256);
```

Not sure if both options are valid, but it would have made my code simpler from the start if I realised I didn't need the callback.

**Comment by hipstersmoothie** [comment]
Yeah I think everywhere we should push people towards this and not callbacks
